training-intern/src/components/exam2/components/DropZoneItem.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div class="container" @drop.prevent="handleDrop">
+    <div
+      class="drop-zone"
+      @dragenter.prevent="toggleAction"
+      @dragover.prevent
+      @dragleave.prevent="toggleAction"
+      @drop.prevent="toggleAction"
+      :class="{ active: isActive }"
+    >
+      <div class="drop-icon">
+        <IconUpload />
+      </div>
+      <div class="drop-description">
+        <h3>{{ placeholder }}</h3>
+        <label for="drop-select" class="text-lable">{{ triggerText }}</label>
+        <input id="drop-select" type="file" @change="handleUpload" multiple />
+      </div>
+    </div>
+    <div class="file-zone">
+      <SelectedFile
+        :files="files"
+        @handleDelete="handleDelete"
+        @handleSubmit="handleSubmit"
+      />
+      <div
+        v-if="msg.error || msg.success"
+        :class="{ success: msg.success, error: msg.error }"
+      >
+        {{ msg.error }}{{ msg.success }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import IconUpload from "@/assets/icon/IconUpload.vue";
+import SelectedFile from "./SelectedFile.vue";
+import {
+  validateDuplicate,
+  validateFileSize,
+  validateExtension,
+  validateLimitExtention,
+  formatBytes,
+} from "@/utils/validate.js";
+import { v4 as uuidv4 } from "uuid";
+import { mapActions } from "vuex";
+
+export default {
+  data() {
+    return {
+      msg: {
+        error: "",
+        success: "",
+      },
+      isActive: false,
+      listFileNames: [],
+    };
+  },
+  props: {
+    placeholder: {
+      type: String,
+      default: () => "Drag and drop files",
+    },
+    triggerText: {
+      type: String,
+      default: () => "Browse files",
+    },
+    files: {
+      type: Array,
+      default: () => [],
+    },
+    uploadMessage: {
+      type: String,
+      default: () => "",
+    },
+    uploadStatus: {
+      type: Boolean,
+      default: () => false,
+    },
+    MAX_SIZE: {
+      type: Number,
+      default: () => 0,
+    },
+    QUANTITY_FILE_UPLOAD: {
+      type: Number,
+      default: () => 0,
+    },
+    LIST_EXTENTIONS: {
```

Comment:
tên các prop thì a đặt theo kiểu camelCase đi đừng đặt in hoa hết như constant thế này, truyền vào thì truyên constant nhưng ten props không phải constant